Runtime/io.github.lyuma.av3emulator.asmdef

```diff
@@ -1,15 +1,15 @@
-{

```

Comment:
I think big diff is caused because of autocrlf setting mismatch. in old (master) file, CRLF is used and I commit as LF.

In my opinion, we should use LF in the git repo and enable autocrlf on windows because it's very common even for windows projects but which should we use?

Reply:
according to https://github.com/lyuma/Av3Emulator/commit/f2dddd921b56fa1d6478f16768b5ca56808385c1, this project looks to use LF.